src/Tribe/PUE/Checker.php

```diff
@@ -1173,5 +1204,55 @@ protected function get_site_domain() {
 				return strtolower( $site_url['host'] );
 			}
 		}
+
+		/**
+         * Returns the plugin file to use when checking for the licensed component or plugin.
+         *
+		 * @param string $plugin_file The component or plugin file in the `<dir>/<file>.php` format.
+		 *
+		 * @return string A component or plugins file in the `<dir>/<file>.php` format.
+		 */
+		protected function get_network_plugin_file( $plugin_file ) {
+		    $map = array(
+		     'event-aggregator/event-aggregator.php' => 'the-events-calendar/the-events-calendar.php',
+            );
+
+		    return isset($map[$plugin_file]) ? $map[$plugin_file] : $plugin_file;
```

Comment:
Some more funky whitespace issues here :dango: 

(Mix of tabs and regular spaces before the first keyword)